src/test/java/test/JmxMetricsTest.java

```diff
@@ -177,4 +179,50 @@ void expose_jmx_metrics_bitbucket_mesh(Product product) throws Exception {
         final var jmxConfigMap = resources.getConfigMap(product.getHelmReleaseName() + "-jmx-config").getDataByKey("jmx-config.yaml");
         assertThat(jmxConfigMap).hasTextContaining("- pattern: \".*\"");
     }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = {"bitbucket"}, mode = EnumSource.Mode.INCLUDE)
+    void service_monitor_bitbucket_mesh(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "monitoring.serviceMonitor.create", "true",
+                product + ".mesh.enabled", "true"
+        ));
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
/nit This relies on the default `3`. To be safe, I would add `.mesh.replicaCount = 3` to the values. It makes it easier to understand why we check 3.

Reply:
I don't think we'll change the default but yes, setting replicaCount is the right thing to do.